apps/builder/app/canvas/shared/styles.ts

```diff
@@ -54,18 +70,28 @@ const helperStyles = [
   }`,
 ];
 
+const subscribePreviewMode = () => {
+  return isPreviewModeStore.subscribe((isPreviewMode) => {
+    if (isPreviewMode) {
+      helpersCssEngine.clear();
+      helpersCssEngine.render();
```

Comment:
Maybe instead set some media query as rendering condition?

Reply:
Like will something like "not all" work as condition?

Reply:
Yeah thought about it too, maybe better? idk, doesn't really matter as long as this is small